active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox)
+
+## Is this the right control? 
+
+Use a **NumberBox** control to recieve and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across order of operations on entered equations. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
+```
+
+### Add placeholder text and a header
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+### Enable calculation support
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+### Add formatting
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP addresss" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000"
+    FormatType="IPAddress" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for international telephone" 
+    Header="Enter equation:" 
+    PlaceholderText="0 (123) 456-7890"
+    FormatType="InternationalTelephone" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for custom input" 
+    Header="Enter equation:" 
+    PlaceholderText="00 ft 00 in"
+    CustomFormatString="## ft ## in"
+    StripLeadingZeros="False" />
+```
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    Text="2"
+    StepSize="2"
+    UpDownArrowsEnabled="True"
+    UpDownDragEnabled="True" />
+```
+
+### Add a prefix and suffix 
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP address" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000" 
+    Prefix="IP Address:" />
+```
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for cents" 
+    Header="Enter equation:" 
+    PlaceholderText="0.00" 
+    Suffix="cents" />
+```
+
+### Add mininum and maximum values
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    PlaceholderText="1" 
+    MinValue="1"
+    MaxValue="99" />
+```
+
+## Remarks
```

Comment:
It's not explicitly called out anywhere, but I assume the control will expose the entered (or calculated) value via the `Text` property provided from the underlying TextBox control. 
- If this is the case it will be important to call this out as it would mean that a scenario where a `Quantity` property that is specified as an int in a ViewModel and is bound to a `NumberBox` control in the View would require a converter. (Or are there, currently undocumented, plans to provide conversion for scenarios like this to be built into the control?
- If this is not the case, then this needs explaining.

Reply:
Great question. NumberBox's main input field will be the Text property that comes off of its parent class, TextBox. We have made considerations that storing an easily accessible Double, Value, beside Text would be a convenience for developers. 

Effectively, Text and Value should always match (aside from being a conversion of each other) and when one is changed, events will be fired as it is validated before the opposite is updated. I have defined the current order and firing times of these events in API Notes > Behavioral Components > Events. 

I believe we have a more up-to-date thread on this in your review further down but please let me know if there are any absences of clarity or anything you would change!